### PR TITLE
XIVY-15138 Remove Typebrowser-Combobox & Move Combine-Icon

### PR DESCRIPTION
**Pull request by ivy-edp**
After consulting with Bruno, I’m removing the combobox from the "Add Attribute" dialog. I initially added it to allow quicker type selection without opening the browser. However, according to Bruno, this additional combobox causes more confusion than benefit, so I’m removing it. In the near future, I plan to improve keyboard navigation for the browser.
Now it looks like this:
grafik

Additionally, I’ve moved the "Combine" icon to the right and made it always visible (except in Neo). Like the delete button, it will now simply be disabled if combining is not possible.
grafik

### CODE REVIEW

**Comment by ivy-edp** [comment]
@ivy-lli fyi

File: packages/dataclass-editor/src/master/DataClassMasterContent.tsx [hunk 1]
```diff
@@ -171,6 +162,16 @@ export const DataClassMasterContent = () => {
         disabled={table.getSelectedRowModel().rows.length === 0}
         aria-label='Delete field'
       />
+      {!context.file.includes('/neo') && (
```
Comment (ivy-lgi) [inline packages/dataclass-editor/src/master/DataClassMasterContent.tsx:165]: Now that it's always visible, I especially vote for having a separator between the buttons.

File: packages/dataclass-editor/src/master/DataClassMasterContent.tsx [hunk 2]
```diff
@@ -171,6 +162,16 @@ export const DataClassMasterContent = () => {
...
+        <Button
+          key='combineButton'
+          icon={IvyIcons.WrapToSubprocess}
+          onClick={() => combineFields.mutate()}
+          aria-label='Combine fields'
+          title='Combine fields'
+          disabled={table.getSelectedRowModel().rows.length <= 0}
```
Comment (ivy-lgi) [inline packages/dataclass-editor/src/master/DataClassMasterContent.tsx:172]: I don't think the length of an array can be smaller than 0, right?

**Review by ivy-lgi — APPROVED** [review]

File: packages/dataclass-editor/src/master/DataClassMasterContent.tsx (hunk 1 above)
Comment (ivy-edp) [reply packages/dataclass-editor/src/master/DataClassMasterContent.tsx:165]: Somthing like this?
grafik

I feel like there might be a few too many separators then. If there’s already a separator between each icon, why would we even need them at all? But we can also take a quick look at this together in the office (maybe with Anais) if you’d like.

**Comment by ivy-lli** [comment]
Really sad that this is gone...
Don't understand why this should be more complicated than the full type browser. IMO this was a really nice fast way to give users a quick way to add new attributes, which especially @ivy-bb should like 🤷‍♂️

**Pull request merged**